cogs/info.py

```diff
@@ -44,13 +44,10 @@ async def trivia (self, ctx, arg1 = "", arg2 = ""):
         # *** ANSWER ***
         elif (arg1.lower() == "answer"):
             if (self.bot.trivia.check(arg2) == 0):
-                await ctx.send("Correct")
-                # Handle what should happen when a correct
-                # answer is submitted
+                await ctx.send(f"Answer: " + arg2 + " is correct!")
```

Comment:
This is an incomplete implementation. In addition to informing the user that they submitted a correct answer, there needs to be code that ends the current Trivia question and either ends the game or moves on to the next question.

In addition, for the notification of a correct answer, I think it would be good to say who it was that submitted the correct answer.